lib/command-runner.js

```diff
@@ -15,10 +16,23 @@ CommandRunner.prototype.run = function(path, args, opts, message) {
 
 function doRun(runner, path, args, opts, message) {
   return new Promise(function(resolve, reject) {
-    var options = opts || {cwd: runner.sitePath, stdio: 'inherit'},
+    var command,
+        options = opts || {cwd: runner.sitePath},
         msg = message || 'rebuild failed for';
 
-    childProcess.spawn(path, args, options).on('close', function(code) {
+    command = childProcess.spawn(path, args, options);
+
+    command.stdout.setEncoding('utf8');
+    command.stdout.on('data', function(data) {
+      runner.logger.log(data);
+    });
+
+    command.stderr.setEncoding('utf8');
+    command.stderr.on('data', function(data) {
+      runner.logger.error(data);
+    });
```

Comment:
I don't _think_ so, because `runner.logger` is an instance of `BuildLogger` from `lib/build-logger.js`, and isn't a stream. It sends content both to the console and to the log file for the current build.